opendps/uui_number.c

```diff
@@ -129,12 +129,13 @@ static void number_draw(ui_item_t *_item)
         break;
     }
     uint16_t xpos = _item->x - w;
+    uint32_t color = item->color;
```

Comment:
This should be a uint16_t I believe

Reply:
Done. Fixed up a couple of build warnings in the process, too.